packages/flutter/test/widgets/table_test.dart

```diff
@@ -932,5 +932,28 @@ void main() {
     },
   );
 
+  testWidgets(
+    'Table widget - Default textBaseline is set to TableBaseline.alphabetic',
+    (WidgetTester tester) async {
+      await tester.pumpWidget(
```

Comment:
indent seems to be off

Reply:
Thanks for the review. I addressed the indent and other minor styling issues (like adding trailing `,`) and pushed.